portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,24 +32,26 @@
 import java.util.*;
 
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns user comments. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
     // Private class to represent comments as objects in Datastore.
     private class Comment {
         private long id;
         private String text;
+        private long time;
 
-        public Comment(long id, String text) {
+        public Comment(long id, String text, long time) {
```

Comment:
Please add a comment something like this that describes the arguments to this constructor.
(I assume we aren't going to do javadoc)

```
/* Class Comment.
  id -- a unique identifier from ?DataStore?.
  text -- the comment
  time_millis -- timestamp, System time in milliseconds
*/
```

Reply:
Thanks, done.